internal/common/config/config.go

```diff
@@ -58,3 +70,57 @@ type DisasterConfig struct {
 	SpatialPDFType         shared.SpatialPDFType // Set x,y prob. distribution of the disaster's epicentre (more post MVP)
 	MagnitudeLambda        float64               // Exponential rate param for disaster magnitude
 }
+
+// ForagingConfig captures foraging-specific config
+type ForagingConfig struct {
+	DeerHuntConfig DeerHuntConfig
+	FishingConfig  FishingConfig
+}
+
+// GameConfig returns the configuration of the game.
+// (Made a function so it cannot be altered mid-game).
+func GameConfig() Config {
+	deerConf := DeerHuntConfig{
```

Comment:
GameConfig is acquired from flags now, this should not be here.

You can pass the configs as required from the server (`s.gameConfig....`)

Reply:
Ah yeah thanks